content/committees/_index.md

```diff
@@ -10,5 +10,8 @@ layout: "list-landing-page"
 This document provides a short summary of the roles and responsibilities of Jakarta EE Working Group Committees 
 during Jakarta EE releases.
 
+* [Guidelines for jakarta.ee website maintainer](/documents/committees/gudelines-jakartaee-website-maintainers.pdf) \
```

Comment:
There's no reason for gudelines-jakartaee-website-maintainers to be a pdf file.
Convert it to a markdown file.  It uses very simple formatting.  Also, it has links
to two other documents that don't seem to be working when that page is viewed
on GitHub.

Reply:
@bshannon let me look into make the change per your suggestion

Reply:
This is now a page:
https://deploy-preview-678--jakartaee.netlify.com/documents/website-guidelines/